src/pyfmi/fmi.pyx

```diff
@@ -3266,6 +3363,10 @@ cdef class FMUModelME1(FMUModelBase):
         """
         Deallocate memory allocated
         """
+        self._invoked_dealloc = 1
+        if self._log_stream:
```

Comment:
Same here, this will make it impossible to log to the stream in the terminate / free methods.

Reply:
Fixed as well (see above)